src/main/java/com/sequenceiq/cloudbreak/controller/json/HostGroupAdjustmentJson.java

```diff
@@ -24,4 +25,16 @@ public Integer getScalingAdjustment() {
     public void setScalingAdjustment(Integer scalingAdjustment) {
         this.scalingAdjustment = scalingAdjustment;
     }
+
+    public Boolean isWithStackUpdate() {
+        return withStackUpdate;
+    }
+
+    public void setWithStackUpdate(Boolean withStackUpdate) {
+        this.withStackUpdate = withStackUpdate;
+    }
+
+    public Boolean getWithStackUpdate() {
```

Comment:
We have a 'isWithStackUpdate()' function as well to get value of withStackUpdate field.